deployment/cfn/data_plane.py

```diff
@@ -0,0 +1,545 @@
+                                      FromPort=p, ToPort=p)
+                for p in [GRAPHITE_WEB, KIBANA, SSH]
+            ] + [
+                ec2.SecurityGroupRule(IpProtocol='tcp',
+                                      CidrIp=VPC_CIDR,
+                                      FromPort=p, ToPort=p)
+                for p in [GRAPHITE, RELP, STATSITE]
+            ] + [
+                ec2.SecurityGroupRule(IpProtocol='udp', CidrIp=VPC_CIDR,
+                                      FromPort=p, ToPort=p)
+                for p in [STATSITE]
+            ],
+            SecurityGroupEgress=[
+                ec2.SecurityGroupRule(IpProtocol='tcp',
+                                      CidrIp=VPC_CIDR,
+                                      FromPort=p, ToPort=p)
+                for p in [POSTGRESQL, REDIS, SSH]
+            ] + [
+                ec2.SecurityGroupRule(IpProtocol='tcp',
+                                      CidrIp=ALLOW_ALL_CIDR,
+                                      FromPort=p, ToPort=p)
+                for p in [HTTP, HTTPS]
+            ],
+            Tags=self.get_tags(Name=bastion_security_group_name)
+        ))
+
+        bastion_host_name = 'BastionHost'
+
+        return self.add_resource(ec2.Instance(
+            bastion_host_name,
+            BlockDeviceMappings=[
+                {
+                    "DeviceName": "/dev/sda1",
+                    "Ebs": {
+                        "VolumeType": "gp2",
+                        "VolumeSize": "256"
+                    }
+                }
+            ],
+            InstanceType=Ref(self.bastion_instance_type_parameter),
+            KeyName=Ref(self.keyname_parameter),
+            ImageId=Ref(self.bastion_host_ami_parameter),
+            NetworkInterfaces=[
+                ec2.NetworkInterfaceProperty(
+                    Description='ENI for BastionHost',
+                    GroupSet=[Ref(bastion_security_group)],
+                    SubnetId=Select("0", Ref(self.public_subnets)),
+                    AssociatePublicIpAddress=True,
+                    DeviceIndex=0,
+                    DeleteOnTermination=True
+                )
+            ],
+            Tags=self.get_tags(Name=bastion_host_name)
+        ))
+
+    def create_rds_instance(self):
+        rds_security_group_name = 'sgDatabaseServer'
+
+        rds_security_group = self.add_resource(ec2.SecurityGroup(
+            rds_security_group_name,
+            GroupDescription='Enables access to database servers',
+            VpcId=Ref(self.vpc_id),
+            SecurityGroupIngress=[
+                ec2.SecurityGroupRule(
+                    IpProtocol='tcp', CidrIp=VPC_CIDR, FromPort=p, ToPort=p
+                )
+                for p in [POSTGRESQL]
+            ],
+            SecurityGroupEgress=[
+                ec2.SecurityGroupRule(
+                    IpProtocol='tcp', CidrIp=VPC_CIDR, FromPort=p, ToPort=p
```

Comment:
Nope, good catch. Addressed in https://github.com/WikiWatershed/model-my-watershed/commit/ff9e2268465a71a4602eb154d6a6c38cbe676d31.